match_modules.go

```diff
@@ -385,6 +387,84 @@ func (p *Parser) parseTcp(f *map[string]Flag) (state, error) {
 	return sStart, nil
 }
 
+func (p *Parser) parseMultiport(f *map[string]Flag) (state, error) {
+	s := sStart
+	for tok, lit := p.scanIgnoreWhitespace(); tok != EOF; tok, lit = p.scanIgnoreWhitespace() {
+		for nextValue := false; !nextValue; {
+			nextValue = true
+			switch s {
+			case sStart:
+				switch tok {
+				case NOT:
+					s = sINotF
+				case FLAG:
+					s = sIF
+					nextValue = false
+				default:
+					return sError, fmt.Errorf("unexpected token %q, expected flag, or \"!\"", lit)
+				}
+			case sINotF:
+				switch {
+				case lit == "--dports" || lit == "--destination-ports":
+					(*f)["destination-ports"] = Flag{
+						Not:    true,
+						Values: p.parsePorts(),
+					}
+					s = sStart
+				case lit == "--sports" || lit == "--source-ports":
+					(*f)["source-ports"] = Flag{
+						Not:    true,
+						Values: p.parsePorts(),
+					}
+					s = sStart
+				case lit == "--ports":
+					(*f)["ports"] = Flag{
+						Not:    true,
+						Values: p.parsePorts(),
+					}
+					s = sStart
+				default:
+					// The end of the match statement is reached.
+					// Since we already scanned the ! charackter,
+					// we have to return the sNot state, or
+					// unscanIgnoreWhitespace twice (this can fail
+					// because of a fixed sized buffer, that is full
+					// of Whitespaces).
```

Comment:
```suggestion
```

I think we can remove the comment as we already return the `sNot` State. I comment is wrong at the moment. (probably at other places too)